### DISCUSSION TITLE
Contributos, what's your workflow for developing and PRing contributions?

### DISCUSSION THREAD

**Discussion by gcjbr** [laravel/framework#51042] [Q&A]
I find it hard to contribute to Laravel core because the code itself is not something I can run as a Laravel application.

What I usually do is running a brand new Laravel app, modifying the core straight there, and when it works modify the actual forked core I have.

I'm sure this is not the best solution, would anyone care to share how they do it?

**Comment by henzeb** [comment]
I barely PR things. Most of the time it's not going to be merged anyway, so why bother?

**Reply by gcjbr** [reply]
Let's say: when you rarely PR, what's your workflow? :p

**Reply by shaedrich** [reply]
@henzeb Welcome to the club! If one isn't a friend of Taylor, chances are very slim.

**Reply by ghost** [reply]
=)) very encouraging guys :)) I wanted to contribute in few occasions but forking the repository was THE game stopper for me.

But I can tell you that I created macros to create a parallel (modified) flow in my app. So I did not change the vendor files.
